CHANGELOG.md

```diff
@@ -13,6 +13,29 @@ Remember to update the hyperlinks at the bottom.
 
 [//]: # (current developments)
 
+## 25.5.1 (2025-08-04)
+
+### Enhancements
+
+* Bump to `conda 25.5.1`, `conda-libmamba-solver 25.4.0`, `constructor 3.12.1`. (#159)
+
+### Bug fixes
+
+* Set package cache location for tests to a temporary directory to avoid creating a `pkgs` file into the `conda-standalone` location for `onedir` builds. (#160)
```

Comment:
```suggestion
* Set package cache location for tests to a temporary directory to avoid creating a `pkgs` directory into the `conda-standalone` location for `onedir` builds. (#160)
```